pkg/config/config.go

```diff
@@ -996,6 +996,12 @@ func InitConfig(config Config) {
 	config.BindEnv("runtime_security_config.enable_runtime_compiled_constants")
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump_manager.enabled", false)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump_manager.cleanup_period", 30)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump_manager.tags_resolution_period", 60)
```

Comment:
nit: activity_dump instead of activity_dump_manager (sounds a bit too implementation level)

Reply:
👍🏻